modules/apps/notification/notification-web/src/main/java/com/liferay/notification/web/internal/portlet/template/NotificationTemplatesTemplateHandler.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.notification.web.internal.portlet.template;
+
+import com.liferay.notification.constants.NotificationConstants;
+import com.liferay.notification.constants.NotificationPortletKeys;
+import com.liferay.notification.model.NotificationTemplate;
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.kernel.template.BaseTemplateHandler;
+import com.liferay.portal.kernel.template.TemplateHandler;
+import com.liferay.portal.kernel.template.TemplateVariableGroup;
+import com.liferay.portal.kernel.templateparser.TemplateNode;
+import com.liferay.portal.kernel.util.LinkedHashMapBuilder;
+import com.liferay.portal.kernel.util.Portal;
+
+import java.util.Locale;
+import java.util.Map;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Victor Kammerer
+ */
+@Component(
+	property = "javax.portlet.name=" + NotificationPortletKeys.NOTIFICATION_TEMPLATES,
+	service = TemplateHandler.class
+)
+public class NotificationTemplatesTemplateHandler extends BaseTemplateHandler {
```

Comment:
Shouldn't be `NotificationTemplateTemplateHandler` ?

Reply:
Sure!